model/invoice.rb

```diff
@@ -164,6 +164,10 @@ def generate_pdf(data)
       pdf.text "#{data[:issuer_address]},"
       pdf.text "#{data[:issuer_city]}, #{data[:issuer_state]} #{data[:issuer_postal_code]},"
       pdf.text data[:issuer_country]
+      # :nocov:
+      pdf.text "#{data[:issuer_in_eu_vat] ? "VAT" : "Tax"} ID: #{data[:issuer_tax_id]}" if data[:issuer_tax_id]
```

Comment:
I realize it was like this already, but what is the reason behind the `:nocov:` here?  Is this difficult to cover?

Reply:
I added test for it